contrib/nydusify/pkg/packer/packer.go

```diff
@@ -67,7 +69,10 @@ func (cfg *BackendConfig) rawMetaBackendCfg() []byte {
 		"access_key_id":     cfg.AccessKeyID,
 		"access_key_secret": cfg.AccessKeySecret,
 		"bucket_name":       cfg.BucketName,
-		"object_prefix":     cfg.MetaPrefix + "/",
+		"object_prefix":     defaultMetaPrefix,
```

Comment:
How do users put the object in the root directory of bucket?

Reply:
> How do users put the object into the root directory of bucket?

Compatible to specified suffix / now.
User get it by prefix = "/"